RELEASE.md

```diff
@@ -1,3 +1,37 @@
+# Release 1.7.0
+
+The 1.7 minor series tracks TensorFlow 1.7.
+
+## Hightlights
+
+- (Beta) New Beholder plugin that shows a live video feed of tensor data during
+  model training. Developed and contributed by Chris Anderson (@chrisranderson).
```

Comment:
`Developed and contributed by Chris Anderson (@chrisranderson)` could be shortened to `by @chrisranderson`

Reply:
That generates a relative link which doesn't work if viewing this file anywhere other than from the final location (e.g. it wouldn't work when viewed from this pull request page or from a commit view, etc.).  Also, this style of link was used for the last several dashboard intro blurbs further down in this file.

Reply:
Done.